markdown/opentelemetry-intro/opentelemetry-intro.md

```diff
@@ -0,0 +1,320 @@
+
+![logicalarch](img/logicalarch.png)
+
+## Set up a Solace PubSub+ Broker 
+Duration: 0:05:00
+
+Access to a Solace messaging service, Solace PubSub+, can be achieved in either one of the three flavours
+1. Hardware Appliance
+1. Software broker image (Docker, Virtual image)
+1. Solace Cloud service instance
+
+This tutorial will walk you through setting up a Solace Cloud service instance, which also gives you access to the Event Portal. If you are interested in setting up a local broker running on Docker or a virtual machine check out the [PubSub+ Event  Broker: Software](https://docs.solace.com/Solace-SW-Broker-Set-Up/SW-Broker-Image-Setup.htm) documentation
+
+### Sign up for free Solace Cloud account 
+Navigate to the [Create a New Account](https://console.solace.cloud/login/new-account) page and fill out the required information. No credit card required!
+
+### Create a messaging service
+After you create your Solace Cloud account and sign in to the Solace Cloud Console, you'll be routed to the event mesh page.
+
+![Solace Cloud Event Mesh Page](img/landing-page-event-mesh.png "Solace Cloud Event Mesh")
+
+
+Click on 'Messaging Services' and all the messaging services associated with your account will show up if you have any already created. To create a new service, click either button as depicted in the image below:
+
+![Solace Cloud Landing Page](img/landing-page-signup.png "Solace Cloud Landing Page")
+
+Fill out all the details for your messaging service, and then click "Create" at the bottom of the page.
+
+![Create Solace Cloud Messaging Service](img/create-service.png "Solace Cloud Messaging Service")
+
+Your service should be ready to use in a couple seconds! 🌪
+
+Click on the previously created service and navigating to the **Connect** tab. 
+Expand the "Solace Messaging" menu to get the connection details
+![connection-service](img/connection-service.png "connection-service")
+
+Use a text editor to save the the connection parameters with the **Host**, **Message VPN Name**, **Client Username** and **Password**. You'll need them later.
+
+✔️ You've created the first part of the solution!
+![check1](img/check1.png)
+
+## Set up Jaeger All-In-One
+
+Duration: 0:05:00
+
+Jaeger collects and visualizes the spans created by your microservices.  It has several different components such as an underlying database, a GUI and a span collector, but fortunately it has an all-in-one distribution that makes setup easy.
+
+Negative
+: If you don't have a tool that opens .gz files, you can download one. [7-Zip](https://www.7-zip.org/download.html) is a good free option
+
+### Steps to install
+1. Extract the Jaeger distribution file you downloaded as part of the prerequisites.  (e.g. `jaeger-1.17.1-windows-amd64.tar.gz` on Windows)
+1. Open a command terminal.
+1. Run `jaeger-all-in-one`
+1. Using a web browser, navigate to `http://localhost:16686/`
+1. You should be greeted by a picture of a...chipmunk?
```

Comment:
I think its a Gopher! hahah its its similar to [Go's Gopher](https://www.google.com/search?q=go+gopher&rlz=1C5CHFA_enCA891CA891&oq=go+gopher&aqs=chrome.0.0l2j46j0l5.956j0j7&sourceid=chrome&ie=UTF-8)